checklists/alz_checklist.en.json

```diff
@@ -916,17 +917,6 @@
             "training": "https://learn.microsoft.com/learn/modules/design-implement-azure-expressroute/",
             "link": "https://learn.microsoft.com/azure/expressroute/designing-for-disaster-recovery-with-expressroute-privatepeering#challenges-of-using-multiple-expressroute-circuits"
         },
-        {
```

Comment:
I'm not sure why we would remove this.  Is there a reason we would not want the domain controllers to be in the more fault-tolerant configuration?

Reply:
I think this is being consolidated into B03.11

Reply:
I was reading B03.11 as being just for Entra Domain Services - I think this might be tied to my other comment thread then.

I am thinking the sheet should be agnostic to the provider used, and we can treat Active Directory Domain Services and Entra Domain Services with the same recommendations - the same HA for DCs applies to both.  Although I may be missing some roadmap items here.

Reply:
Correct, consolidating under the identity design area